modules/retail/pages/retail-deploy-terminals-auto.adoc

```diff
@@ -63,7 +81,47 @@ autosign_grains:
 
 
 
-.Procedure: Configuring Saltboot to Auto-Sign During PXE Boot
+// For SLE12 and SLE11 templates, the procedure is the following:
+.Procedure: Configuring Saltboot to Keep Auto-Signed Grains (SLE 11 and SLE 12)
+. In the location where the image source is built, such as a build host or source repository, create a configuration file called [path]``etc/salt/minion.d/autosign-grains.conf``.
+  This must be outside of the root directory provided by the template.
```

Comment:
```suggestion
  This must be outside of the [path]``root`` directory provided by the template.
```

Template comes with `root` directory, so we are talking literally about directory named root next to where config.xml is located.